### PR TITLE
#16 Replaced string.Format with FormatWith usage

### PR DESCRIPTION
**Pull request by baradgur**
---
name: Fix for #16
about: Wrong message error displayed
---

**Description**
Replaced string.Format with FormatWith usage

**PR Domain**
Replaced string.Format with FormatWith usage

- [ ] Add feature
- [x] Fix bug
- [ ] Fix Typo
- [ ] Extend support

**Issue relation**
This PR is related to issue #16

**Screenshots**
'No Screenshots'

### CODE REVIEW

File: Source/EyesGuard/Views/Pages/CustomPause.xaml.cs [hunk 1]
```diff
@@ -47,27 +48,27 @@ private void PauseProtection_Click(object sender, RoutedEventArgs e)
...
-                    warning += string.Format("» " + App.LocalizedEnvironment.Translation.EyesGuard.TimeManipulation.MinutesLimit, 59);
+                    warning += "» " + App.LocalizedEnvironment.Translation.EyesGuard.TimeManipulation.MinutesLimit.FormatWith(new { Minutes = 59 });
                 }
 
                 if (seconds > 59)
                 {
                     if (warning != "")
                         warning += "\n";
-                    warning += string.Format("» " + App.LocalizedEnvironment.Translation.EyesGuard.TimeManipulation.SecondsLimit, 59);
+                    warning += "» " + App.LocalizedEnvironment.Translation.EyesGuard.TimeManipulation.SecondsLimit.FormatWith(new { Minutes = 59 });
```
Comment (0xaryan) [inline Source/EyesGuard/Views/Pages/CustomPause.xaml.cs]: Shouldn't this be `new { Seconds = 59 }` ?

File: Source/EyesGuard/Views/Pages/Settings.xaml.cs [hunk 2]
```diff
@@ -140,20 +141,20 @@ private void SaveButton_Click(object sender, RoutedEventArgs e)
...
-                    warning += string.Format("» " + App.LocalizedEnvironment.Translation.EyesGuard.TimeManipulation.MinutesLimit, 59);
+                    warning += "» " + App.LocalizedEnvironment.Translation.EyesGuard.TimeManipulation.MinutesLimit.FormatWith(new{ Minutes = 59 });
                 }
 
                 if (sgS > 59 || sdS > 59 || lgS > 59 || ldS > 59)
                 {
                     if (warning != "")
                         warning += "\n";
-                    warning += string.Format("» " + App.LocalizedEnvironment.Translation.EyesGuard.TimeManipulation.SecondsLimit, 59);
+                    warning += "» " + App.LocalizedEnvironment.Translation.EyesGuard.TimeManipulation.SecondsLimit.FormatWith(new { Minutes = 59 });
```
Comment (0xaryan) [inline Source/EyesGuard/Views/Pages/Settings.xaml.cs]: Same for here.

`new { Seconds = 59 }`

**Comment by baradgur** [comment]
oops, fixed it

**Comment by 0xaryan** [comment]
LGTM

**Pull request merged**